benchmarks/inc/udt.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+template <typename Contained>
+struct trivial {
+    Contained c;
+};
+
+template <typename Contained>
+inline bool operator==(const trivial<Contained>& lhs, const trivial<Contained>& rhs) {
+    return lhs.c == rhs.c;
+}
+
+template <typename Contained>
+inline bool operator!=(const trivial<Contained>& lhs, const trivial<Contained>& rhs) {
+    return lhs.c != rhs.c;
+}
+
+template <typename Contained>
+struct non_trivial {
+    Contained c;
+    non_trivial() : c() { /* user provided */
+    }

```

Comment:
We should clang-format `benchmarks/inc` and `benchmarks/src`: https://github.com/microsoft/STL/blob/17fde2cbab6e8724d81c9555237c9a623d7fb954/tools/format/CMakeLists.txt#L34-L39